pkg/batcher/createfleet.go

```diff
@@ -70,7 +70,7 @@ func execCreateFleetBatch(ec2api ec2iface.EC2API) BatchExecutor[ec2.CreateFleetI
 			for _, instanceID := range reservation.InstanceIds {
 				requestIdx++
 				if requestIdx >= len(inputs) {
-					logging.FromContext(ctx).Errorf("received more instances than requested, ignoring instance %s", aws.StringValue(instanceID))
+					log.FromContext(ctx).Error(fmt.Errorf("received more instances than requested, ignoring instance %s", aws.StringValue(instanceID)), "Batcher error")
```

Comment:
I had similar comments on the upstream PR, but I think we should aim to align these with the rest of our logging semantics of noncapitalized
```suggestion
					log.FromContext(ctx).Error(fmt.Errorf("received more instances than requested, ignoring instance %s", aws.StringValue(instanceID)), "batcher error")
```